redaxo/src/addons/be_style/plugins/customizer/boot.php

```diff
@@ -1,77 +1,101 @@
 <?php
-
 /**
  * REDAXO customizer.
  *
  * Codemirror by : http://codemirror.net/
  * Marijn Haverbeke <marijnh@gmail.com>
  */
 
-if (rex::isBackend() && rex::getUser()) {
-    $config = $this->getConfig();
+// Plugin-Config
+$config = rex_plugin::get('be_style', 'customizer')->getConfig();
 
-    rex_view::addCssFile($this->getAssetsUrl('css/styles.css'));
+/* Output CodeMirror-CSS */
+if (rex::isBackend() && rex_request('codemirror_output', 'string', '') == 'css') {
+    rex_response::cleanOutputBuffers();
+    header("Content-type: text/css");
 
-    if ($config['codemirror']) {
-        // Codemirror + Theme-CSS
-        rex_view::addCssFile($this->getAssetsUrl('vendor/codemirror/codemirror.css'));
-        rex_view::addCssFile($this->getAssetsUrl('vendor/codemirror/addon/display/fullscreen.css'));
-        rex_view::addCssFile($this->getAssetsUrl('vendor/codemirror/theme/'.$config['codemirror_theme'].'.css'));
-        if (isset($config['codemirror-tools']) && $config['codemirror-tools']) {
-            rex_view::addCssFile($this->getAssetsUrl('vendor/codemirror/addon/fold/foldgutter.css'));
-        }
+    $filenames = array();
+    $filenames[] = $this->getAssetsUrl('vendor/codemirror/codemirror.css');
+    $filenames[] = $this->getAssetsUrl('vendor/codemirror/addon/display/fullscreen.css');
+    $filenames[] = $this->getAssetsUrl('vendor/codemirror/theme/'.$config['codemirror_theme'].'.css');
+    if (isset($config['codemirror-tools']) && $config['codemirror-tools']) {
+        $filenames[] = $this->getAssetsUrl('vendor/codemirror/addon/fold/foldgutter.css');
+    }
 
-        // Theme
-        if ($config['codemirror_theme'] != '') {
-            rex_view::setJsProperty('customizer_codemirror_defaulttheme', $config['codemirror_theme']);
-        }
+    foreach ($filenames as $filename) {
+        echo '/* ' . $filename . ' */' . "\n" . rex_file::get($filename) . "\n";
```

Comment:
prinzipiell finde ich gut weniger dateien an den browser zu übertragen (vorallem in diesem konkreten fall da es echt viele sein können).

das ganze in dieser art zu lösen hat in meinen Augen folgende Probleme
- wir umgehen jegliches browser caching (etag, last-modified, etc) was wir mit `rex_response->sendXXX()`  hätten
- ich vermute, dass dieses zusammgefasste file kein cachebuster parameter autom. von redaxo bekommt (was vorher pro file der fall war)
--> kein automatisches neuladen der Dateien nach update des addons

wäre super wenn wir das ganze hinbekommen ohne die genannten probleme